src/NetTopologySuite/Operation/Union/UnionStrategy.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using NetTopologySuite.Geometries;
+
+namespace NetTopologySuite.Operation.Union
+{
+    /// <summary>
+    /// An strategy class that allows UnaryUnion to adapt to different
+    /// kinds of overlay algorithms.
+    /// </summary>
+    /// <author>Martin Davis</author>
+    public class UnionStrategy
```

Comment:
this isn't designed to be inherited from, so let's mark it `sealed`

(We can always unseal it later if that becomes a problem)